universe/Ship.cpp

```diff
@@ -590,6 +589,7 @@ void Ship::SetShipMetersToMax() {
         if (max_it == m_part_meters.end())
             continue;
 
+        max_it->second.SetCurrent(Meter::LARGE_VALUE);
```

Comment:
In Planet::Colonize it sets the target population and population to the colony ship population, then backpropegates immediately. Given the greater variety of effects that can control what the initial ship part max capacity and capacity meters should be, it's probably more difficult to determine the correct initial value... but I suppose it doesn't matter as the current should get clamped to the max after the max is set correctly by effects. So I suppose it's fine...